primitives/session-keys/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+authors = ['Manta Network']
+description = "Primitives for session keys"
+edition = "2021"
+license = 'GPL-3.0'
+name = "session-key-primitives"
+version = "1.0.0"
+
+[dependencies]
+manta-primitives = { path = "../manta", default-features = false }
+nimbus-primitives = { git = "https://github.com/manta-network/nimbus.git", branch = "polkadot-v0.9.22", default-features = false }
+parity-scale-codec = { version = "3.0.0", default-features = false, features = ["derive"] }
+scale-info = { version = "2.0", default-features = false, features = ["derive"] }
+sp-application-crypto = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.22", default-features = false }
+sp-consensus-aura = { git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "polkadot-v0.9.22" }
+sp-core = { git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "polkadot-v0.9.22" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.22", default-features = false }
+
```

Comment:
Maybe you should add `runtime-benchmarks` feature in the future. I suppose it should be enabled in the future for new if we benchmark consensus related pallets.

Reply:
It will be enabled in https://github.com/Manta-Network/Manta/pull/582